packages/infolists/resources/views/components/text-entry.blade.php

```diff
@@ -56,8 +62,18 @@
             >
                 <{{ $isListWithLineBreaks ? 'ul' : 'div' }}
                     @class([
+                        'flex flex-wrap items-center' => ! $isListWithLineBreaks,
+                        'gap-1.5' => $isBadge,
                         'list-inside list-disc' => $isBulleted(),
-                        'flex flex-wrap items-center gap-1.5' => $isBadge,
+                        match ($alignment) {
+                            Alignment::Start => 'justify-start',
+                            Alignment::Center => 'justify-center',
+                            Alignment::End => 'justify-end',
+                            Alignment::Justify => 'justify-between',
```

Comment:
This should be `Alignment::Between`, and probably just `null` for `Alignment::Justify`?

Reply:
this is tricky. as the HasAlignment Trait doesn't have a setter for `Alignment::Between`. so you can't configure with `->alignBetween`.

Reply:
https://github.com/filamentphp/filament/blob/642a78e613665ab6f5c6fb1c10b56ef18df02f1f/packages/tables/resources/views/components/columns/column.blade.php#L26-L34

😬

Reply:
There's an `alignment()` method, right?
That would allow people to pass an enum instance.

Reply:
should `alignBetween` be added? or its on purpose not to include

Reply:
Isn't it the same as `justify` in this case?

Reply:
what I meant is in this trait https://github.com/filamentphp/filament/blob/3.x/packages/support/src/Concerns/HasAlignment.php there is no `alignBetween` method

> Isn't it the same as `justify` in this case?

then in this `match` can be updated to `Alignment::Between, Alignment::Justify => 'justify-between text-justify', `


or `alignBetween` method is purposely not provided?

Reply:
Yeah that's what I meant about the match expression.
I think adding an `alignBetween` method makes sense, @danharrin?

Reply:
If it ensures consistency, sure